assets/Components/ShakeEventExpo.js

```diff
@@ -0,0 +1,28 @@
+import { Accelerometer } from 'expo-sensors';
```

Comment:
Nice that you got this to work, its totally fine to use copy pasted code like this when you're trying things out and learning for the first time. If you have time and energy, try to refactor this into a functional component :)